src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,92 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  handleClose: () => void;
```

Comment:
In terms of naming conventions, it would be more intuitive to rename the `handleClose` prop to `onClose` to make it clear that it's an event handler function. Following the naming conventions for event handlers in React can make the code more readable and easier to understand.